RubeesEat.IntegrationTests/SeleniumTests/RubeesEatSeleniumTests.cs

```diff
@@ -0,0 +1,20 @@
+using RubeesEat.IntegrationTests.PageObjects;
+
+namespace RubeesEat.IntegrationTests.SeleniumTests;
+
+[TestFixture]
+public class RubeesEatSeleniumTests : SeleniumIntegrationTestBase
+{
+    public RubeesEatSeleniumTests() : base("UserHomepage")
+    {
+    }
+
+    [Test]
+    public void GetAccountBalance()
+    {
+        var page = Start<UserHomepagePageObject>();
+        Assert.That(
+            page.GetAccountBalance(),
+            Is.EqualTo("+ 1000000,50 \u20ac"));
+    }
```

Comment:
`Is.EqualTo("+ 1000000,50 \u20ac"));` needs to be adapted if you use the currency format